coordinator/persistence-api/src/test/java/io/stargate/db/query/builder/BuiltSelectTest.java

```diff
@@ -69,57 +59,6 @@ public void testSelectColumnsNoWhere() {
     assertThat(names(select.selectedColumns())).isEqualTo(asSet("k2", "v1"));
   }
 
-  @ParameterizedTest
-  @MethodSource("functionsToTest")
-  public void testFunctions(BuiltQuery<?> query, String expectedQueryString) {
-    assertBuiltQuery(query, expectedQueryString, emptyList());
-
-    BoundSelect select = checkedCast(query.bind());
-
-    assertBoundQuery(select, expectedQueryString);
-
-    assertThat(select.isStarSelect()).isFalse();
-    assertThat(names(select.selectedColumns())).isEqualTo(asSet("k2", "v1", "v2"));
-  }
-
-  public static Stream<Arguments> functionsToTest() {
-    Supplier<QueryBuilder.QueryBuilder__20> base = () -> newBuilder().select().column("k2", "v1");
```

Comment:
This is the reason test needs to be removed -- refers to internal names of generated classes, and since generation removed now not easy to convert. But test itself not very valuable due to limited usage of this QueryBuilder variant.